Generator/AssociationTypeGenerator.php

```diff
@@ -21,6 +21,8 @@
  */
 class AssociationTypeGenerator implements GeneratorInterface
 {
+    const TYPE = 'association_types';
+
     const ASSOCIATION_TYPES_FILENAME = 'association_types.csv';
```

Comment:
This is not the scope of that PR and there are lot of things to rewrite. You should create an issue for that.